### PR TITLE
Design updates with Obj/Widget Thumbnails

### PR DESCRIPTION
**Pull request by apepper**

### CODE REVIEW

File: src/Widgets/DataFormBooleanWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/DataFormBooleanWidget/thumbnail.svg:1]: The top left border looks strange. This affects multiple thumbnails.

File: src/Widgets/DataIconConditionWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/DataIconConditionWidget/thumbnail.svg:1]: This is a DataFormWidget, not a FormWidget. Therefor the color should be orange.

File: src/Widgets/CardWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/CardWidget/thumbnail.svg:1]: No text like "Card" in the icon? This effects multiple thumbnails.

File: src/Widgets/BannerHeadlineContainerWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/BannerHeadlineContainerWidget/thumbnail.svg:1]: The thumbnail for "BannerHeadlineWidget" is missing. Please add it.

File: src/Widgets/DataFormDeleteButtonWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/DataFormDeleteButtonWidget/thumbnail.svg:1]: Other button icons do have a mouse pointer over the button. I suggest to also add one mouse pointer here as well. This affects other button widgets as well.

**Review by apepper — COMMENTED** [review]
FEEDBACK ([38b2f05](https://github.com/Scrivito/scrivito-portal-app/pull/204/commits/38b2f0525a60ac976a90184dc903099fab849c77))

Nice start 🚀! The thumbnails look nice!

Please see my comments for next steps. It would also be great if all widget thumbnails could be part of this PR.

File: src/Widgets/ButtonWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/ButtonWidget/thumbnail.svg:1]: There is a grey "on" above the white "on" of the end of the word. The same is true for the Checkout button.

File: src/Widgets/DataIconConditionWidget/thumbnail.svg
Comment (apepper) [inline src/Widgets/DataIconConditionWidget/thumbnail.svg:1]: This is a "Data widget", not a "data form widget". Please remove the "form" text.

File: src/Widgets/LinkContainerWidget/LinkContainerWidgetEditingConfig.ts [hunk 1]
```diff
@@ -2,10 +2,11 @@ import { Link, provideEditingConfig } from 'scrivito'
 import { classNameToThumbnail } from '../../utils/classNameToThumbnail'
 import { LinkWidget } from '../LinkWidget/LinkWidgetClass'
 import { LinkContainerWidget } from './LinkContainerWidgetClass'
+import Thumbnail from './thumbnail.svg'
```
Comment (apepper) [inline src/Widgets/LinkContainerWidget/LinkContainerWidgetEditingConfig.ts:5]: For `LinkContainerWidget` there is already an icon. But there is one missing for `LinkWidget`. Please add it.

File: src/Widgets/SpaceWidget/SpaceWidgetEditingConfig.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import { provideEditingConfig } from 'scrivito'
 import { SpaceWidget } from './SpaceWidgetClass'
-import { classNameToThumbnail } from '../../utils/classNameToThumbnail'
+import Thumbnail from './thumbnail.svg'
 
 provideEditingConfig(SpaceWidget, {
```
Comment (apepper) [inline src/Widgets/SpaceWidget/SpaceWidgetEditingConfig.ts:5]: There is a fairly new widget, which icon is still missing: SearchResultsWidget`. Please add it.

**Review by apepper — COMMENTED** [review]
FEEDBACK ([6c56b7b](https://github.com/Scrivito/scrivito-portal-app/pull/204/commits/6c56b7bd1f4f8aed350476678264a2b8781fec16))

Very nice looking icons 🚀!

There are only some minor details and two or so missing widget icons. Getting closer!

**Review by apepper — COMMENTED** [review]
FEEDBACk ([7573108](https://github.com/Scrivito/scrivito-portal-app/pull/204/commits/75731082c7499f3b454c9153f226524e6bcf57a0))

Almost there! The icons look really nice: 🚀.

Open Todos:
* [x] https://github.com/Scrivito/scrivito-portal-app/pull/204#discussion_r1430643606
* [x] As a FYI: https://github.com/Scrivito/scrivito-portal-app/pull/204#discussion_r1426705399

File: src/Widgets/DataFormDeleteButtonWidget/thumbnail.svg
Comment (agessler) [reply src/Widgets/DataFormDeleteButtonWidget/thumbnail.svg:1]: I just want to leave as it is. I understand the difference. I am not fine with the current state. Maybe I would like to change all data / dataform widget previews as well in future.

First thing we should address is to separating of widgets/data/form widgets in sidebar. The issue for uses is not the missing pointer in preview - its the grouping.

Currentyl we only group `last used` and `all`

File: src/Widgets/DataIconConditionWidget/thumbnail.svg
Comment (agessler) [reply src/Widgets/DataIconConditionWidget/thumbnail.svg:1]: Done with neweest commit

**Comment by agessler** [comment]
https://github.com/agessler/scrivito-portal-app/commit/a6bfd34ef87fadddc6b27c6f792c9b5cccf762f0

**Pull request merged**